src/store/modules/rewards/index.ts

```diff
@@ -241,4 +241,18 @@ export class RewardsModule extends VuexModule.With({
 
     return new BigNumber(shrinkToken(result, 18));
   }
+
+  @action async fetchRewardsMultiplier({
+    poolId,
+    reserveId
+  }: {
+    poolId: string;
+    reserveId: string;
+  }): Promise<BigNumber> {
+    const result = await this.contract.methods
+      .rewardsMultiplier(this.currentUser, poolId, reserveId)
+      .call();
+
+    return new BigNumber(shrinkToken(result, 6));
```

Comment:
Why is the decimals hard coded to 6?

Reply:
Haven't found a decimal returned from the contract
0xB443DEA978B39178Cb05Ae005074227A4390DfCe

Reply:
I don't think we're going to find any decimals from that contract, what token is ```result``` in?